.github/workflows/run-playwright-tests.yml

```diff
@@ -32,6 +32,14 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '18.x'
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+      - name: Build docker container
+        run: ./gradlew jibDockerBuild -x test -x checkstyleMain -x checkstyleTest --image odd-platform:e2e-latest-build
       - name: Start containers
```

Comment:
It looks like we will have a new prerequisite for running tests.

@DementevNikita, Could u pls mention `Gradle` in the list of prerequisites in README: 
https://github.com/opendatadiscovery/odd-platform/blob/main/tests/README.md
?

Reply:
renamed to `Run tests`